.github/sample-test-config.yaml

```diff
@@ -0,0 +1,28 @@
+# Test configuration file for running performance benchmarks
```

Comment:
Please change the name of this file to something other than "sample". If this is being used for integration tests, please indicate this.